app/presenters/file_presenter.rb

```diff
@@ -1,9 +1,4 @@
 # app/presenters/file_presenter.rb
 class FilePresenter < Sufia::GenericFilePresenter
-  self.terms = [:dc_type, :title, :creator, :contributor, :description,
-                :tag, :rights, :publisher, :date, :subject, :language,
-                :spatial, :temporal, :abstract, :tableOfContents, :format, 
-		:bibliographicCitation, :provenance, :isReferencedBy, :relation,
-		:rights, :title, :isCitedBy, :isIdenticalTo, :isPartOf,
-		:isVersionOf, :doi, :hdl, :orcid, :rid]
+  self.terms = ScholarsArchiveSchema.properties.map(&:name)
```

Comment:
It also has the properties that are inherited from Sufia. I would expect that `terms =` would leave a bunch of missing stuff from your form (like title, for instance)